test/recipes/80-test_ssl_old.t

```diff
@@ -576,6 +576,22 @@ sub testssl {
             ok(run(test(['ssl_old_test', '-psk', '0102030405', '-cipher', '@SECLEVEL=2:DHE-PSK-AES128-CCM'])),
                'test auto DH meets security strength');
           }
+
+        SKIP: {
+            skip "skipping auto ECDH PSK tests", 1
+                if ($no_ec || $no_psk);
+
+            ok(run(test(['ssl_old_test', '-psk', '0102030405', '-cipher', '@SECLEVEL=3:ECDHE-PSK-AES256-CBC-SHA384'])),
+               'test auto DH meets security strength');
+          }
+
+        SKIP: {
+            skip "skipping auto DH PSK tests", 1
+                if ($no_dh || $no_psk);
+
+            ok(run(test(['ssl_old_test', '-psk', '0102030405', '-cipher', '@SECLEVEL=3:DHE-PSK-AES128-CCM'])),
+               'test auto DH meets security strength');
+          }
```

Comment:
- Without specifying `-tls1_2` or earlier, this test is not really covering the new code: if I understand this recipe correctly, the default is equivalent to `-tls1_3` if nothing is specified. We should explicitly test for `-tls1_2`, not sure about previous versions.
- Maybe we could add also negative tests here: testing that the RSA PSK ciphers are not accepted at SECLEVEL=3.
- Also maybe we should fill the gap and add tests to also cover the `SSL_kDHE` and `SSL_kECDHE` in TLS < 1.3

Reply:
@mattcaswell you are more familiar with these tests than I am, can you check what I commented here?

Reply:
> Without specifying -tls1_2 or earlier, this test is not really covering the new code: if I understand this recipe correctly, the default is equivalent to -tls1_3 if nothing is specified. We should explicitly test for -tls1_2, not sure about previous versions.

Yes - this is correct. Good catch.

Reply:
Too good catch - it doesn't work :)